example/tests/test_deferred.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+from django.test import TestCase
+from shop import deferred
+
+import six
+
+
+def create_regular_class(name, fields={}, meta={}):
+    meta.setdefault('app_label', 'foo')
+    Meta = type(b'Meta', (), meta)
```

Comment:
please do not drop ``from __future__ import unicode_literals ``
Instead use  ``Meta = type(str('Meta'), (), meta)``
or even better, create
```
def create_regular_class(name, fields={}, meta={}):
    class Meta:
        meta = ...
```
and use that